src/extension/packagerStatusIndicator.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import {window, Disposable, StatusBarItem, StatusBarAlignment} from "vscode";
+
+/**
+ * Updates the Status bar with the status of React Native Packager.
+ */
+
+export enum PackagerStatus {
+    PACKAGER_STARTED = 0,
+    PACKAGER_STOPPED
+}
+
+export class PackagerStatusIndicator implements Disposable {
+    private packagerStatusItem: StatusBarItem;
+    private static PACKAGER_STARTED_STATUS_STR: string = "React Native Packager: Running";
+    private static PACKAGER_STOPPED_STATUS_STR: string = "React Native Packager: Stopped";
+
+    public constructor() {
+        this.packagerStatusItem = window.createStatusBarItem(StatusBarAlignment.Left);
+    }
+
+    public dispose(): void {
+        this.packagerStatusItem.dispose();
+    }
+
+    public updatePackagerStatus(status: PackagerStatus): void {
+        switch (status) {
+            case PackagerStatus.PACKAGER_STARTED:
+                this.packagerStatusItem.text = `$(briefcase) ${PackagerStatusIndicator.PACKAGER_STARTED_STATUS_STR}`;
```

Comment:
At the moment you are calling updatePackagerStatus( with parameters specified on the call site. I think it'd be simpler to just have two methods updatePackagerStatusToStarted() updatePackagerStatusToStopped() rather than this switch.